src/sentry/static/sentry/app/components/group/suggestedOwners.jsx

```diff
@@ -190,6 +208,9 @@ class SuggestedOwners extends React.Component {
             <GuideAnchor target="owners">
               <h6>
                 <span>{t('Ownership Rules')}</span>
+                <Hovercard body={this.getHovercardBody()} containerClassName="pill-icon">
+                  <InlineSvg src="icon-circle-info" size="16px" />
```

Comment:
We have new icon styles: `<IconInfo size="xs" />`
https://storybook.getsentry.net/branches/master/index.html?path=/story/style-icons--icon-set